packages/instantsearch.css/src/components/autocomplete.scss

```diff
@@ -0,0 +1,329 @@
+  --ais-autocomplete-primary-color-alpha: 0.2;
+  --ais-autocomplete-muted-color-rgb: 128, 126, 163;
+  --ais-autocomplete-muted-color-alpha: 0.6;
+
+  // Input
+  --ais-autocomplete-search-input-height: 44px;
+  --ais-autocomplete-input-icon-size: 20px;
+
+  // Size and spacing
+  --ais-autocomplete-base-unit: 16;
+  --ais-autocomplete-spacing-factor: 1;
+  --ais-autocomplete-spacing: calc(var(--ais-autocomplete-base-unit) * var(--ais-autocomplete-spacing-factor) * 1px);
+  --ais-autocomplete-spacing-half: calc(var(--ais-autocomplete-spacing) / 2);
+  --ais-autocomplete-panel-max-height: 650px;
+
+  // Z-index
+  --ais-autocomplete-base-z-index: 9998;
+
+  // Font
+  --ais-autocomplete-font-size: calc(var(--ais-autocomplete-base-unit) * 1px);
+  --ais-autocomplete-font-family: inherit;
+  --ais-autocomplete-font-weight-medium: 500;
+  --ais-autocomplete-font-weight-semibold: 600;
+  --ais-autocomplete-font-weight-bold: 700;
+
+  // Icons
+  --ais-autocomplete-icon-size: 20px;
+  --ais-autocomplete-icon-stroke-width: 1.6;
+  --ais-autocomplete-icon-color-rgb: 119, 119, 163;
+  --ais-autocomplete-icon-color-alpha: 1;
+  --ais-autocomplete-action-icon-size: 20px;
+
+  // Border colors
+  --ais-autocomplete-panel-border-color-rgb: 128, 126, 163;
+  --ais-autocomplete-panel-border-color-alpha: 0.3;
+  --ais-autocomplete-input-border-color-rgb: 128, 126, 163;
+  --ais-autocomplete-input-border-color-alpha: 0.8;
+
+  // Background colors
+  --ais-autocomplete-background-color-rgb: 255, 255, 255;
+  --ais-autocomplete-background-color-alpha: 1;
+  --ais-autocomplete-input-background-color-rgb: 255, 255, 255;
+  --ais-autocomplete-input-background-color-alpha: 1;
+  --ais-autocomplete-selected-color-rgb: 179, 173, 214;
+  --ais-autocomplete-selected-color-alpha: 0.205;
+  --ais-autocomplete-description-highlight-background-color-rgb: 245, 223, 77;
+  --ais-autocomplete-description-highlight-background-color-alpha: 0.5;
+
+  // Detached mode
+  --ais-autocomplete-detached-media-query: (max-width: 680px);
+  --ais-autocomplete-detached-modal-media-query: (min-width: 680px);
+  --ais-autocomplete-detached-modal-max-width: 680px;
+  --ais-autocomplete-detached-modal-max-height: 500px;
+  --ais-autocomplete-overlay-color-rgb: 115, 114, 129;
+  --ais-autocomplete-overlay-color-alpha: 0.4;
+
+  // Shadows
+  --ais-autocomplete-panel-shadow: 0 0 0 1px rgba(35, 38, 59, 0.1),
+    0 6px 16px -4px rgba(35, 38, 59, 0.15);
+
+  // Scrollbar
+  --ais-autocomplete-scrollbar-width: 13px;
+  --ais-autocomplete-scrollbar-track-background-color-rgb: 234, 234, 234;
+  --ais-autocomplete-scrollbar-track-background-color-alpha: 1;
+  --ais-autocomplete-scrollbar-thumb-background-color-rgb: var(--ais-background-color-rgb);
+  --ais-autocomplete-scrollbar-thumb-background-color-alpha: 1;
+
+  // Touch screens
+  @media (hover: none) and (pointer: coarse) {
+    --ais-autocomplete-spacing-factor: 1.2;
+    --ais-autocomplete-action-icon-size: 22px;
+  }
+}
```

Comment:
I think we need to reuse or update the common CSS variables in `packages/instantsearch.css/src/shared/_variables.scss`. We can also add autocomplete-specific CSS variables to this file, as I did for the chat-specific ones.